packages/paste-core/core-bundle/rollup.package-list.js

```diff
@@ -0,0 +1,37 @@
+export const rollupFileList = [
+  'src/absolute.tsx',
+  'src/alert.tsx',
+  'src/anchor.tsx',
+  'src/aspect-ratio.tsx',
+  'src/avatar.tsx',
+  'src/box.tsx',
+  'src/button.tsx',
+  'src/card.tsx',
+  'src/combobox.tsx',
+  'src/combobox-primitive.tsx',
+  'src/disclosure.tsx',
+  'src/disclosure-primitive.tsx',
+  'src/flex.tsx',
+  'src/form.tsx',
+  'src/grid.tsx',
+  'src/heading.tsx',
+  'src/list.tsx',
+  'src/media-object.tsx',
+  'src/menu.tsx',
+  'src/menu-primitive.tsx',
+  'src/modal.tsx',
+  'src/modal-dialog-primitive.tsx',
+  'src/non-modal-dialog-primitive.tsx',
+  'src/paragraph.tsx',
+  'src/popover.tsx',
+  'src/screen-reader-only.tsx',
+  'src/separator.tsx',
+  'src/spinner.tsx',
+  'src/stack.tsx',
+  'src/tabs.tsx',
+  'src/tabs-primitive.tsx',
+  'src/text.tsx',
+  'src/tooltip.tsx',
+  'src/tooltip-primitive.tsx',
+  'src/truncate.tsx',
```

Comment:
Can this be a glob?
`src/*.tsx`

Reply:
Sadly, no 👎 
`[!] Error: Could not resolve entry module (src/*.tsx).`